src/sentry/conf/server.py

```diff
@@ -1445,6 +1445,7 @@ def create_partitioned_queues(name):
 SENTRY_METRICS_INDEXER = "sentry.sentry_metrics.indexer.postgres_v2.StaticStringsIndexerDecorator"
 SENTRY_METRICS_INDEXER_OPTIONS = {}
 SENTRY_METRICS_INDEXER_CACHE_TTL = 3600 * 2
+SENTRY_METRICS_INDEXER_WRITES_LIMITER_OPTIONS = {}
```

Comment:
I'm not against this change but if you think it would be cleaner (or use-case specific) you could put this config in https://github.com/getsentry/sentry/blob/master/src/sentry/sentry_metrics/configuration.py#L18-L24